### PR TITLE
crypto/rsa/rsa_pss.c: silence coverity warning

### PR DESCRIPTION
**Pull request by mspncp**
Reported by Coverity Scan (CID ~~1439136~~ 1439138)

### CODE REVIEW

File: crypto/rsa/rsa_pss.c [hunk 1]
```diff
@@ -244,7 +244,7 @@ int RSA_padding_add_PKCS1_PSS_mgf1(RSA *rsa, unsigned char *EM,
 
  err:
     EVP_MD_CTX_free(ctx);
-    OPENSSL_clear_free(salt, sLen);
+    OPENSSL_clear_free(salt, (size_t)sLen); /* salt != NULL implies sLen > 0 */
```
Comment (mspncp) [inline crypto/rsa/rsa_pss.c]: See https://github.com/openssl/openssl/pull/7156/commits/c4911ecbd774b43215de41627fb302befb01c7a9#diff-d900cab16b5cf32fa7e71c4f94665310R196.

**Comment by mspncp** [comment]
Giving the ci bots another try..

**Comment by mspncp** [comment]
The single [Travis job failure](https://travis-ci.org/openssl/openssl/jobs/426532013) is unrelated (caused by the MarkInfo problem).

**Comment by mattcaswell** [comment]
Pushed.

**Pull request closed without merging**